### PR TITLE
Update quota on files upload

### LINKED ISSUE #6690: Quota progress bar should update on delete/upload
capture d ecran_2017-09-27_16-48-13

On upload/deletion, the quota bar doesn't update. It could be a small change but a great addition! :)
@nextcloud/designers @nextcloud/javascript .

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #6690
capture d ecran_2017-09-27_16-48-13

### CODE REVIEW

File: apps/files/templates/appnavigation.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ class="nav-icon-<?php p($item['icon'] !== '' ? $item['icon'] : $item['id']) ?> s
 			</a>
 		</li>
 		<?php } ?>
-		<li id="quota" class="pinned <?php
+		<li id="quota" class="pinned <?php p($pinned===0?'first-pinned ':'') ?><?php
```
Comment (skjnldsv) [inline apps/files/templates/appnavigation.php]: This fixes the pinned position of the quota if the files_trashbin app is disabled (meaning no other apps have registered another app-navigation pinned entry)

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Review by rullzer — APPROVED** [review]
:+1: awesome!

**Review by rullzer — DISMISSED** [review]
jsunit tests fail

**Comment by skjnldsv** [comment]
Build failure unrelated.

**Review by rullzer — APPROVED** [review]

**Comment by skjnldsv** [comment]
Apparently there is a race condition sometimes where the quota status get requested before the MOVE request gets executed. Which result in an incomplete update. I'll fix it.

**Pull request merged**